README.md

```diff
@@ -53,18 +42,11 @@ The Eclipse repositories for this plugin.
 Release update site.
 
 * [Eclipse Marketplace entry for the GWT Plugin](https://marketplace.eclipse.org/content/gwt-plugin)
-* [Update sites for the GWT Plugin](https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/)
+* [Update sites for the GWT Plugin](https://gwt-plugins.github.io/documentation/repo/site/current/)
```

Comment:
I'm really not sure that github a pages is a good place for storing eclipse plugins